packages/cos/build.yaml

```diff
@@ -131,3 +131,38 @@ excludes:
 - ^/run/.*
 
 {{ end }}
+{{- end}}
+#### system/cos system/cos-container and recovery/cos end here
+
+#### recovery/cos-img starts here
+{{ if eq .Values.name "cos-img" }}
+requires:
```

Comment:
maybe this requires block can be shared with `cos-squash`?

Reply:
it can but...do we want to? Like I rather have 5 duplicated lines if it means one less if statement. I kind of like the current state of having the 3 different templates one after another instead of mangling everything with ifs.